src/main/java/com/google/api/codegen/config/DiscoInterfaceModel.java

```diff
@@ -75,15 +77,21 @@ public DiscoApiModel getApiModel() {
   @Override
   public List<MethodModel> getMethods() {
     ImmutableList.Builder<MethodModel> methods = ImmutableList.builder();
-    if (!apiModel.getDocument().resources().containsKey(interfaceName)) {
+    if (!apiModel.getDocument().resources().containsKey(getResourceName())) {
+      System.out.println(getSimpleName());
       return ImmutableList.of();
     }
-    for (Method method : apiModel.getDocument().resources().get(interfaceName)) {
+    for (Method method : apiModel.getDocument().resources().get(getResourceName())) {
       methods.add(new DiscoveryMethodModel(method, null));
     }
     return methods.build();
   }
 
+  private String getResourceName() {
```

Comment:
what does this do?

Reply:
Converts the interface name to a resource name. The format of the interface name is `[ownerName].[documentName].[version].[resourceName]`.